src/appleseed/foundation/math/sampling/qmcsamplingcontext.h

```diff
@@ -265,6 +319,8 @@ inline void QMCSamplingContext<RNG>::split_in_place(
     m_sample_count = sample_count;
     m_instance = 0;
 
+    m_cmj_params = CMJParameters(sample_count, dimension, m_base_instance, m_mode);
```

Comment:
I'm not quite sure to understand how this can work given that in the vast majority of the cases, sample_count is just 1 (grep for `split_in_place` in the code). In that cases CMJ degenerates to pure random no?

Overall I doubt that CMJ is a good fit for our style of general purpose sampler. Maybe it would make sense to use it only for image plane sampling.

Reply:
Yes, I intended to use it only on image plane sampling but at the time i wasn't sure on how to achieve this. I think starting over isn't a bad idea. 

What do you think of the idea of having 2 different samplers ?

Reply:
up @dictoon